docs/topics/testing/tools.txt

```diff
@@ -2188,7 +2188,7 @@ redirected into a ``StringIO`` instance::
         def test_command_output(self):
             out = StringIO()
             call_command("closepoll", stdout=out)
-            self.assertIn("Expected output", out.getvalue())
+            self.assertIn("Successfully closed poll", out.getvalue())
```

Comment:
Can you also add `poll_ids` to the `call_command()` call?

Reply:
Passing `poll_ids=[1]` should work:
```
call_command("closepoll", poll_ids=[1], stdout=out)
```

Reply:
You can add more commits to this branch.

Reply:
okay

Reply:
Does this mean we should have another test example without another argument in the call command? We might lose the simplicity of the example if we add another argument. Any thoughts on that?

Reply:
> Does this mean we should have another test example without another argument in the call command? We might lose the simplicity of the example if we add another argument. Any thoughts on that?

Another example is not necessary, IMO. This won't be "abstract" anymore so even with `poll_ids` is simple enough.